public/auth.html

```diff
@@ -3,8 +3,8 @@
 <title>Login to AVF Engagement & Media Dashboard</title>
     <link rel="shortcut icon" type="image/png" href="images/favicon.png"/>
     <link rel="stylesheet" type="text/css" href="css/styles.css">
-    <script src="vendor/firebase-7.21.1/firebase-app.js"></script>
-    <script src="vendor/firebase-7.21.1/firebase-auth.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/8.0.0/firebase-app.js"></script>
```

Comment:
The other reason for avoiding third-party hosted libs is that we now depend on the 3rd party to always keep the link up, I would still prefer us hosting the libraries and updating them regularly. What do you think?

Reply:
Since the platform is developed by google and also the fact that the older libraries are still being hosted, I'm confident the links will always be up if that is the issue.

Reply:
Thanks this also reduces our server's traffic and makes parallel loading possible.

Reply:
Thanks this also reduces our server's traffic and makes parallel loading possible.